ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeBasemap/main.cpp

```diff
@@ -42,7 +42,7 @@ int main(int argc, char *argv[])
 
```

Comment:
Are we missing the code to set the API key?

Reply:
Saving for a later date as it's not necessary atm.